packages/table/src/table-row-context.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from 'react'
+
+const {
+  Provider: TableRowProvider,
+  Consumer: TableRowConsumer
+} = React.createContext(32)
```

Comment:
If this is meant to provide styling-specific mechanics, consider using "SmashingThemeProvider" instead.